### PR TITLE
Add best python crash course, Please merge this

### PR DESCRIPTION
**Pull request by CyberSecByte**
## What does this PR do?
Add resource

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by CyberSecByte** [comment]
@eshellman have a look here and please merge it

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -378,12 +378,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### Bash
 
 * [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/) - M. Cooper (HTML)
+* [Bash Cook book](https://drive.google.com/file/d/15LVUS7KI6JB9OmZSrkNiwN1DLHaz8ttS/view) - Carl Albing & JP Vossen (PDF)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:381]: Drive links are not supported. Please provide official website links where this resource can be accessed

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -974,6 +974,7 @@
 * [Python Built in Functions A to Z Tutorial and Examples](https://www.youtube.com/playlist?list=PLrJGwAG1U62RW_hGGnk7xXG0LciHkT54X) - Brainy Things
 * [Python Course](https://www.python-course.eu)
 * [Python Course from scratch](https://scrimba.com/playlist/pNpZMAB) - Olaf Paulson (scrimba)
+* [Python Crash course - Python Full Course for all beginners](https://youtu.be/f79MRyMsjrQ) - Programming with mosh
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:977]:
```suggestion
* [Python Crash course - Python Full Course for all beginners](https://www.youtube.com/watch?v=f79MRyMsjrQ) - Programming with mosh
```

no short link

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (CyberSecByte) [reply books/free-programming-books-langs.md:381]: Btw here's the link https://www.pdfdrive.com/bash-cookbook-solutions-and-examples-for-bash-users-e184776614.html also if you don't wanna merge the book, then please merge only course

File: courses/free-courses-en.md (hunk 2 above)
Comment (CyberSecByte) [reply courses/free-courses-en.md:977]: Oh okay can you please merge it now?

**Comment by CyberSecByte** [comment]
ok so i did changes that were requested, please check and merge it

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (LuigiImVector) [reply books/free-programming-books-langs.md]: remove the books

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -392,6 +394,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml) - Paul Armstrong (HTML)
 * [Introduction to Bash Scripting](https://github.com/bobbyiliev/introduction-to-bash-scripting) - Bobby Iliev (Markdown, PDF)
 * [Introduction to the Command Line](https://launchschool.com/books/command_line) - Launch School (HTML)
+* [Linux Shell Scripting](https://drive.google.com/file/d/15Mzn059VjYX92pfwi9zgdo4g54h43H9q/view) - Ken O. Burtch (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:397]: remove this

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -378,12 +378,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 
 * [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/) - M. Cooper (HTML)
+* [Bash Cook book](https://www.pdfdrive.com/bash-cookbook-solutions-and-examples-for-bash-users-e184776614.html) - Carl Albing & JP Vossen (PDF)
 * [Bash Guide for Beginners (2008)](http://www.tldp.org/LDP/Bash-Beginners-Guide/html/) - M. Garrels (HTML)
 * [Bash Notes for Professionals](http://goalkicker.com/BashBook/) - Compiled from StackOverflow documentation (PDF)
 * [BASH Programming (2000)](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO.html) - Mike G. (HTML)
 * [Bash Reference Manual](http://www.gnu.org/software/bash/manual/bashref.html) (HTML)
 * [Bash tutorial](https://web.archive.org/web/20180328183806/http://gdrcorelec.ups-tlse.fr/files/bash.pdf) - Anthony Scemama (PDF)
 * [BashGuide](http://mywiki.wooledge.org/BashGuide) - Maarten Billemont (HTML) [(PDF)](http://s.ntnu.no/bashguide.pdf)
+* [Command Line Kung Fu](https://drive.google.com/file/d/15H5IrItcch2RBYgW78W8ch-oqKdh5KbS/view) - Jason Cannon (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:388]: remove this

File: books/free-programming-books-langs.md [hunk 5]
```diff
@@ -378,12 +378,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### Bash
 
 * [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/) - M. Cooper (HTML)
+* [Bash Cook book](https://www.pdfdrive.com/bash-cookbook-solutions-and-examples-for-bash-users-e184776614.html) - Carl Albing & JP Vossen (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:381]: remove this

**Comment by CyberSecByte** [comment]
I have done the changes and removed those gdrive links

**Comment by LuigiImVector** [comment]
@davorpa `check-urls` lasted more than 6 hours

File: books/free-programming-books-langs.md (hunk 5 above)
Comment (LuigiImVector) [reply books/free-programming-books-langs.md:381]: @CyberSecByte

Comment (CyberSecByte) [reply books/free-programming-books-langs.md:381]: Ok I'm gonna close this and make a new pull request

**Comment by CyberSecByte** [comment]
i'm closing this pr and will make a new one instead

**Pull request closed without merging**